http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -1259,7 +1272,10 @@ class GKEInterpreter[F[_]](
       // Persistence
       raw"""persistence.size=${disk.size.gb.toString}G""",
       raw"""persistence.gcePersistentDisk=${disk.name.value}""",
-      raw"""env.swaggerBasePath=$proxyPath"""
+      raw"""env.swaggerBasePath=$proxyPath""",
+      // cromwellConfig
+      raw"""cromwellConfig.gcsProject=${cluster.googleProject.value}""",
+      raw"""cromwellConfig.gcsBucket=$gcsBucket"""
```

Comment:
During the PR process for https://github.com/broadinstitute/cromwhelm/pull/15 this got simplified to just `config`, so these are now mismatching what the helm chart expects

```suggestion
      raw"""config.gcsProject=${cluster.googleProject.value}""",
      raw"""config.gcsBucket=$gcsBucket"""
```

Reply:
I also had to add `/cromwell-execution` onto the end. Despite being called "bucket", this actually wants a path within the bucket... 🤦

Reply:
(or alternatively, I guess the helm chart could always append something like `/cromwell-execution` to the end of the bucket that Leo provides it with... that's probably better in the long term but would require an additional helm change... 😬